### PR TITLE
Adds new Entity icons

### PR DESCRIPTION
**Pull request by Graham187K**
Adds the remaining known entity icons to Magna-React.

This is the mapping used in Incidents repo: https://github.com/advthreat/incident-manager/blob/master/src/utils/icons.ts

### CODE REVIEW

File: svg/app-window.svg
Comment (androche) [inline svg/app-window.svg:1]: 🥹  aw can we make this our mascot?

File: svg/circuitry.svg
Comment (androche) [inline svg/circuitry.svg:1]: Screenshot 2024-12-02 at 3 06 23 PM

Sometimes we have to target more to get it all the same color, but sometimes its intended. I'm not sure what its supposed to look like, but if you did want it to be all one color we can add the name to the list starting line 214 in `AIcon.scss`

File: svg/seal-question.svg
Comment (androche) [inline svg/seal-question.svg:1]: Screenshot 2024-12-02 at 3 13 08 PM

These could be added to the "AIcon.scss" file listed above to the dot to pick up theming.

File: framework/components/AIcon/magnaIcons.js
Comment (androche) [inline framework/components/AIcon/magnaIcons.js:1]: According to that map it looks like we might be missing: `folder-notch-open`

**Review by androche — COMMENTED** [review]
Looks like you got them all! I think I just found one missing and a couple notes on the color differences that might need at least 2 updates.

File: framework/components/AIcon/magnaIcons.js
Comment (Graham187K) [reply framework/components/AIcon/magnaIcons.js:1]: Good eye! I should have added this detail: I think that's a bug that I noticed while doing this. I'll submit a fix to Incident manager to change it to `folder-open`.

Here's the Figma for icons which calls for `folder-open` on `file_path`'s: 
https://www.figma.com/design/WzioFqYjVBdO3EXRctNZya/Icons%3A-Observables-and-Assets?node-id=659-191&node-type=frame&m=dev

There's actually no `folder-notch-open` in Phosphor, so I'm not sure how that string came to be in code.

File: svg/app-window.svg
Comment (Graham187K) [reply svg/app-window.svg:1]: LOL, now all I can see it a pair of googly eyes when I look at it.

File: svg/circuitry.svg
Comment (Graham187K) [reply svg/circuitry.svg:1]: Ah thank you, I didn't catch that... I need to enlarge the icons more on my screen to check these things in future.

File: svg/seal-question.svg
Comment (Graham187K) [reply svg/seal-question.svg:1]: Done, also added `app-window` and the two dots needed it. I think that's all of them.

File: framework/components/AIcon/magnaIcons.js
Comment (androche) [reply framework/components/AIcon/magnaIcons.js:1]: Oh weird! :) I wonder what a "notch-open" would have looked like 😄

**Review by androche — APPROVED** [review]

**Pull request merged**